frontend/packages/dev-console/src/components/pipelineruns/__tests__/PipelineRunDetailsPage.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { PipelineRunModel } from '../../../models';
+import PipelineRunDetailsPage from '../PipelineRunDetailsPage';
+
+const pipelineRunDetailsProps: React.ComponentProps<typeof PipelineRunDetailsPage> = {
+  match: {
+    isExact: true,
+    path: `/k8s/ns/:ns/${referenceForModel(PipelineRunModel)}/:name/`,
+    url: 'k8s/ns/tekton-pipelines/tekton.dev~v1alpha1~PipelineRun/simple-pipeline-6wjqqw',
+    params: {
+      ns: 'tekton-pipelines',
+    },
+  },
+  kind: 'PipelineRun',
+};
+
+const expectedTabs: string[] = [`Overview`, `YAML`, `Logs`];
+
+describe('PipelineRun details page tests', () => {
+  it('should render `DetailsPage` Component with proper page tabs -> [ `Overview` , `YAML` , `Logs` ]', () => {
+    const pipelineRunDetailsWrapper = shallow(
+      <PipelineRunDetailsPage {...pipelineRunDetailsProps} />,
+    );
+    const pages = pipelineRunDetailsWrapper.find('DetailsPage').prop('pages');
```

Comment:
`.prop('pages')` digs a bit into the DetailsPage API thus testing it, but I think it's the bare minimum to test that we are indeed rendering the tabs we want. I wish there was a less tying fashion we could use but I can't think of anything that would be as useful of a test. I think it's okay, it's unlikely that DetailsPage will change its API.